docs/preview/logdecorations.md

```diff
@@ -0,0 +1,238 @@
+
+# Log decorations
+
+
+Task authors should be able to control how the log output is displayed to the end user.
+This outlines different decoration options that are available.
+
```

Comment:
decorations we had today:
```
##[section] helloworld
##[debug] helloworld
##[error] helloworld
##[warning] helloworld
##[command] helloworld
```

Reply:
yeah good point, we should mention them in the same doc.

Reply:
added